lib/blockscore/actions/update.rb

```diff
@@ -27,20 +32,39 @@ module Update
 
       def_delegators 'self.class', :endpoint, :patch
 
+      # May be deprecated in future releases, prefer :update.
       def save!
         if persisted?
-          patch("#{endpoint}/#{id}", filter_params)
+          patch(update_url, filter_params)
           true
         else
           super
         end
       end
 
+      def update(params)
+        update!(params)
+        true
+      rescue
```

Comment:
Ah well that shouldn't have made it through review. I have [noted it before](https://github.com/BlockScore/blockscore-ruby/pull/20#discussion_r30776192) though